tests/pass-dep/libc/gettid.rs

```diff
@@ -0,0 +1,15 @@
+//@only-target-linux
+
+use libc::{getpid, gettid};
+use std::thread;
+
+fn main() {
+    // Test that in isolation mode a deterministic value will be returned.
+    // The value 1001 is not important, we only care that whatever the value
+    // is, won't change from execution to execution.
+    thread::spawn(|| assert_eq!(unsafe { gettid() }, 1001));
+
+    // Test that the thread ID of the main thread is the same as the process
+    // ID.
+    assert_eq!(unsafe { gettid() }, unsafe { getpid() });
```

Comment:
Please also test that it is *different* from what we got in the child thread.

Reply:
done.